### ISSUE TITLE
Feature suggestion: No index and expiration date

### ISSUE THREAD

**Issue by KlausEverWalkingDev** [SpEcHiDe/AnyDLBot#13]
You could implement those little features as a suggestion to the project :)

**Comment by SpEcHiDe** [comment]
I did not understand, this. 

Sorry. but i might be having trouble understnading what you meant. Could you try explaining in layman terms so that I can grasp it better?

**Comment by KlausEverWalkingDev** [comment]
Sorry, buddy. I was aiming fill a issue to another repo. I don't even remember how I fell down here. I'm closing this issue. Sorry for your trouble. 😌
